### PR TITLE
New LED patterns

### PR DESCRIPTION
**Pull request by legoguy1000**
Added the below patterns that I found from https://www.tweaking4all.com/hardware/arduino/adruino-led-strip-effects/

- Fire
- Bouncing Balls
- Meteor (doesn't really work the same)
- Twinkle

### CODE REVIEW

**Review by owenflatman — APPROVED** [review]
Looks good 👍

**Review by owenflatman — APPROVED** [review]
Looks good 👍

File: simgui.json [hunk 1]
```diff
@@ -2,7 +2,13 @@
   "HALProvider": {
     "Addressable LEDs": {
       "0": {
+<<<<<<< HEAD
```
Comment (gvaldez7206) [inline simgui.json:5]: merge conflicts

**Review by gvaldez7206 — APPROVED** [review]
looks good!!!!!!!

**Pull request merged**